Dockerfile.mysql

```diff
@@ -0,0 +1,10 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=1234
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
+
+EXPOSE 3306
+
+VOLUME /var/lib/mysql
```

Comment:
Add Eof(End-of-file), need to add empty line at the end
![image](https://github.com/mate-academy/devops_todolist_docker_core_task_2_volumes/assets/87720583/6361c87d-c70b-455a-ac11-cde7f074f044)